src/tables.rs

```diff
@@ -177,23 +177,24 @@ pub mod identifier_status {
         '\u{2dd6}'), ('\u{2dd8}', '\u{2dde}'), ('\u{3005}', '\u{3007}'), ('\u{3041}', '\u{3094}'),
         ('\u{3095}', '\u{3096}'), ('\u{3099}', '\u{309a}'), ('\u{309d}', '\u{309e}'), ('\u{30a0}',
         '\u{30a0}'), ('\u{30a1}', '\u{30fe}'), ('\u{3105}', '\u{312c}'), ('\u{312d}', '\u{312d}'),
-        ('\u{312e}', '\u{312e}'), ('\u{312f}', '\u{312f}'), ('\u{31a0}', '\u{31b7}'), ('\u{31b8}',
```

Comment:
this change is strange. According to https://www.unicode.org/Public/security/12.1.0/IdentifierStatus.txt  , \u{312e} is indeed allowed.